pydrawise/client.py

```diff
@@ -270,3 +283,36 @@ async def delete_zone_suspension(self, suspension: ZoneSuspension) -> None:
             id=suspension.id
         ).select()
         await self._mutation(selector)
+
+    async def get_sensors(
+        self,
+        controller: Controller,
+        flow_summary_time_range: FlowSummaryTimeRange | None = None,
+    ) -> list[Sensor]:
+        """Retrieves sensors associated with the given controller.
+
+        :param controller: Controller whose sensors to fetch.
+        :param flow_summary_time_range: Optional time range to query water flow data for.
```

Comment:
The original water_flow method is not exposed. We won't request water flow information unless you specify an explicit time range. I find the default one week not very useful. I want to ultimately be able to create separate sensors in home assistant to track daily, weekly, monthly, and yearly water usage.

We can make it a separate method if you prefer. Not sure that will make things much easier though.

Reply:
What if we make the argument a `timedelta` that applies against `now()` instead? Then the default would remain 1h but the use-case of 1d, 1w, 1m, 1y is fairly simple. That would also make the single method for this be more intuitive without passing an explicit start/end.

Reply:
I don't think we should artificially restrict the API. Even though it would make my use case of 1d, 1w, etc. simple, it would prevent other use cases. E.g., how much water did I use this month vs the same month a year ago?